### DISCUSSION TITLE
10.8.9

### DISCUSSION THREAD

**Discussion by joshuaboniface** [jellyfin/jellyfin#9153] [Announcements]
# Jellyfin 10.8.9

Stable hotfix release for 10.8.z release branch.

## New Features and Major Improvements

N/A

## Release Notes

N/A

## Changelog

GitHub Project: https://github.com/orgs/jellyfin/projects/29

### [jellyfin](https://github.com/jellyfin/jellyfin) [7]

Note: Dependabot automatic PRs are excluded from this list.

 * #9145 [@nyanmisaka] Fix PGS position issue in sw decoding #8602
 * #9130 [@Shadowghost] Fix API access permissions for LiveTV and downloads
 * #9112 [@nyanmisaka] Backport the ffmpeg link fix (#8901) to 10.8.z
 * #9051 [@dmitrylyzo] Fix transcode reasons
 * #9050 [@nyanmisaka] Update workaround for i915 hang in linux 5.18 to 6.1.3
 * #9049 [@Shadowghost] Add more codecs requiring ffmpeg strict -2
 * #9009 [@dmitrylyzo] Fix secondary audio

### [jellyfin-web](https://github.com/jellyfin/jellyfin-web) [9]

Note: Dependabot automatic PRs are excluded from this list.

 * jellyfin/jellyfin-web#4288 [@dmitrylyzo] Fix loading Spanish (Latin America) (es-419)
 * jellyfin/jellyfin-web#4274 [@dmitrylyzo] Fix back action on Hisense TV
 * jellyfin/jellyfin-web#4269 [@thornbill] Fix XSS vulnerability in plugin repo pages
 * jellyfin/jellyfin-web#4267 [@dmitrylyzo] Fix AudioContext limit exceeded
 * jellyfin/jellyfin-web#4263 [@dmitrylyzo] Fix change audio track
 * jellyfin/jellyfin-web#4243 [@dmitrylyzo] Fix HTML escaping in MediaSession and on remote page
 * jellyfin/jellyfin-web#4240 [@dmitrylyzo] Fix detection of SecondaryAudio support
 * jellyfin/jellyfin-web#4238 [@thornbill] Fix XSS vulnerability in router
 * jellyfin/jellyfin-web#4182 [@nyanmisaka] Fix the progressive mp4 transcoding profile

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin/releases/tag/v10.8.9'>10.8.9</a>.</em>

**Comment by Zinoujoker** [comment]
Nice
